### DISCUSSION TITLE
Improper addition of host to our RMM

### DISCUSSION THREAD

**Discussion by NO-TE-Ivan** [amidaware/tacticalrmm#2082] [Q&A]
Good Morning,
I have a problem after installation of tactical RMM.
After installing a test envairoment of tactical RMM, next day with found on RMM console several unwanted hosts, added by some never created users. How can we fix this problem? We attach a sample image. 

![Problem Extra Host](https://github.com/user-attachments/assets/8db25a19-45b6-4279-863f-7a9db1047278)

**Comment by wh1te909** [comment]
https://docs.tacticalrmm.com/faq/#help-ive-been-hacked-and-there-are-weird-agents-appearing-in-my-tactical-rmm
